src/test/java/uk/gov/digital/ho/hocs/info/api/CaseTypeSchemaResourceTest.java

```diff
@@ -0,0 +1,87 @@
+package uk.gov.digital.ho.hocs.info.api;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import uk.gov.digital.ho.hocs.info.domain.exception.ApplicationExceptions;
+
+import java.util.List;
+import java.util.ArrayList;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CaseTypeSchemaResourceTest {
+
+    @Mock
+    private CaseTypeSchemaService caseTypeSchemaService;
+
+    private CaseTypeSchemaResource caseTypeSchemaResource;
+
+    private final static String TEST_CASE_TYPE = "Test";
+
+    @Before
+    public void setUp() {
+        caseTypeSchemaResource = new CaseTypeSchemaResource(caseTypeSchemaService);
+    }
+
+    @Test
+    public void shouldReturnStagesForValidCaseType() {
+        when(caseTypeSchemaService.getCaseTypeStages(TEST_CASE_TYPE)).thenReturn(getMockStageTypes(5));
+
+        ResponseEntity<List<String>> response =
+                caseTypeSchemaResource.getCaseTypeStages(TEST_CASE_TYPE);
+
+        verify(caseTypeSchemaService, times(1)).getCaseTypeStages(eq(TEST_CASE_TYPE));
```

Comment:
times(1) is the default so not needed

Reply:
I was just trying to be explicit, I should've checked really...